windowsservercore-insider/Dockerfile

```diff
@@ -1,11 +1,18 @@
 # escape=`
-FROM mcr.microsoft.com/windows/insider:10.0.18362.113
-
+FROM mcr.microsoft.com/windows/servercore/insider:10.0.19035.1
 
 RUN powershell -Command `
     Add-WindowsFeature Web-Server; `
     Invoke-WebRequest -UseBasicParsing -Uri "https://dotnetbinaries.blob.core.windows.net/servicemonitor/2.0.1.6/ServiceMonitor.exe" -OutFile "C:\ServiceMonitor.exe"
 
+RUN powershell -Command `
+    New-Item -ItemType Directory C:\LogMonitor; `
+    Invoke-WebRequest -UseBasicParsing -Uri "https://github.com/microsoft/windows-container-tools/releases/download/v1.0/LogMonitor.exe" -OutFile "C:\LogMonitor\LogMonitor.exe"; `
+    Invoke-WebRequest -UseBasicParsing -Uri "https://raw.githubusercontent.com/microsoft/iis-docker/master/windowsservercore-insider/LogMonitorConfig.json" -OutFile "C:\LogMonitor\LogMonitorConfig.json"
```

Comment:
Can all powershell invocation be consolidated into 1 script block of the first powershell invocation? If so, it might be more readable to have the urls in a array and have a loop that goes through the array and runs invoke-webrequest on each url. Also can we download all artifacts into 1 folder or does servicemonitor.exe have to be in the root dir?

Reply:
We will consider the file location later. Not this time. I have updated to use a loop as suggested.